docs/proposals/03-networking-extensibility.md

```diff
@@ -11,7 +11,7 @@ Gardener is an open-source project that provides a nested user model. Basically,
 
 For the first set of users, the choice of network plugin might not be so important, however, for the second class of users (i.e., Hosted) it is important to be able to customize networking based on their needs.
 
-Furthermore, Gardener provisions clusters on different cloud-providers with different networking requirements. For example, Azure does not support Calico Networking [1], this leads to the introduction of manual exceptions in static add-on charts which is error prone and can lead to failures during upgrades.
+Furthermore, Gardener provisions clusters on different cloud-providers with different networking requirements. For example, Azure does not support Calico overlay networking with IP in IP [1], this leads to the introduction of manual exceptions in static add-on charts which is error prone and can lead to failures during upgrades.
```

Comment:
Can please update also https://github.com/gardener/gardener/blob/master/docs/extensions/network.md.

Reply:
I've updated it as well!